### PR TITLE
everclear static data

### PR DESCRIPTION
**Pull request by EvgeniiaVak**

### CODE REVIEW

File: protocols/everclear.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "logo_dark": "/protocols/everclear-dark.png",
+    "logo_light": "/protocols/everclear-light.png",
+    "icon_url": "/protocols/everclear-dark.png",
+    "icon_dark": "/protocols/everclear-dark.png",
+    "icon_light": "/protocols/everclear-light.png",
+    "description": "Everclear is a novel public good mechanism for settling intents across chains.",
+    "mainnet": "yes",
+    "type": "bridge",
+    "chains_supported": "Enables permissionless expansion to new chains and rollups, making it possible for any application on any chain to onboard users from any other chain.",
+    "xchain_sync": "Canonical brdiges for message passing",
```
Comment (konradstrachan) [inline protocols/everclear.json:42]: spelling

Comment (konradstrachan) [reply protocols/everclear.json:42]: Also is this true?

File: protocols/everclear.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "auction_design": "RTB",
+    "auction_design_ranking": "Good",
+    "auction_design_details": "Race to the Bottom. Auctioneer smart contract governs filler selection (by honoring the bid with the lowest fees).",
+    "auction_openness": "Good",
```
Comment (konradstrachan) [inline protocols/everclear.json:46]: What would we need for this to be excellent? If everyone can see everything always then I think this doesn't get any better?

**Review by konradstrachan — CHANGES_REQUESTED** [review]
Small comments

File: protocols/everclear.json (hunk 1 above)
Comment (EvgeniiaVak) [reply protocols/everclear.json:42]: > "xchain_sync": "Canonical brdiges for message passing", 
oh, yes, I forgot to ask what is this field anyways? I didn't find anything useful by googling. I just copied that one from the other protocols. I also didn't find it used anywhere on the intent.markets, do we even need this field?

Comment (EvgeniiaVak) [reply protocols/everclear.json]: typo fixed, if it's true or not I don't know yet

File: protocols/everclear.json (hunk 2 above)
Comment (EvgeniiaVak) [reply protocols/everclear.json]: fixed

**Review by konradstrachan — APPROVED** [review]

**Pull request merged**